websites/D/Discord SFX/presence.ts

```diff
@@ -3,10 +3,12 @@ const presence = new Presence({
 });
 
 presence.on("UpdateData", async () => {
-	const presenceData: PresenceData = {
+	const assets = {
+			Logo: "https://cdn.rcd.gg/PreMiD/websites/D/Discord%20SFX/assets/logo.png",
+		},
```

Comment:
For consistency with other presences, let's put this in an enum like so:
https://github.com/PreMiD/Presences/blob/1f5a4117f92c6549a75e1c01c6305f9c4398b5b5/websites/A/Aodesu/presence.ts#L6-L14